extensions/amp-video/0.1/amp-video.js

```diff
@@ -86,8 +85,112 @@ class AmpVideo extends AMP.BaseElement {
     /** @private {!../../../src/mediasession-helper.MetadataDef} */
     this.metadata_ = EMPTY_METADATA;
 
-    /** @private @const {!Array<!UnlistenDef>} */
-    this.unlisteners_ = [];
+    /** @private {?./mixins.AmpVideoMixinDef} */
```

Comment:
could you add some comments on what mixins are and why they are needed for future readers? Maybe have a summary in mixin.js and then put a comment here for reader to see the summary in the other file.